funnel/views/profile.py

```diff
@@ -133,7 +133,7 @@ def view(self):
             .all()
         )
         draft_projects = [
-            proj for proj in self.obj.draft_projects if proj.current_roles.admin
+            proj for proj in self.obj.draft_projects if proj.current_roles.profile_admin
```

Comment:
Change to: `draft_projects = self.obj.draft_projects if self.obj.current_roles.profile_admin else []`